src/components/DocumentationTopic/Summary/Availability.vue

```diff
@@ -122,7 +122,11 @@ export default {
   fill: var(--badge-color);
 
   .theme-dark & {
-    fill: var(--badge-color);
+    fill: var(--badge-dark-color);
+  }
+
+  @include prefers-dark {
+    fill: var(--badge-dark-color);
   }
 }
```

Comment:
Could we add the 
```css
.theme-dark & {
```
class into the `prefers-dark` mixin to not duplicate the code?
Is there any case where we want them to have different properties?